composer.json

```diff
@@ -23,7 +23,7 @@
         "typo3/kickstart": "~3.0.0",
         "typo3/buildessentials": "3.0.x-dev",
         "flowpack/behat": "dev-master",
-        "phpunit/phpunit": "~4.8 || ~5.2.0",
+        "phpunit/phpunit": "~4.8 || ~5.4",
```

Comment:
would advise against a loose dependency, tried that and just a matter of time before it breaks again..